src/pharmpy/tools/iivsearch/tool.py

```diff
@@ -324,6 +382,39 @@ def post_process(rank_type, cutoff, strictness, input_model_entry, base_model_na
         strictness=strictness,
     )
 
+    if rank_type == "mbic":
```

Comment:
How come this is needed here? Is it not possible to just set `n_predicted`, `n_expected` etc if it's mBIC, `None` otherwise? Was thinking so we don't need to double maintain.

Reply:
Moved the option to have it a bit more straightforward. However, the n_predictied and n_expected differ if running bottom up vs running top down. The bottom up approach uses a value different from the default which is the reason for it being different.